example/readmitted.ipynb

```diff
@@ -0,0 +1,260 @@
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {
+    "tags": []
+   },
+   "source": [
+    "### Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.\n",
+    "### SPDX-License-Identifier: Apache-2.0"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "( Run <code>jupyter notebook</code> under the project directory )"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "\n",
+    "import sys\n",
+    "sys.path.append('third-party')\n",
+    "# sys.path.append('../')\n",
+    "\n",
+    "from ppxgboost import BoosterParser as boostparser\n",
+    "from ppxgboost import PPBooster as ppbooster\n",
+    "from ppxgboost import PaillierAPI as paillier\n",
+    "from ppxgboost.PPBooster import MetaData\n",
+    "from ppxgboost.PPKey import PPBoostKey\n",
+    "import sys\n",
+    "import random\n",
+    "import time\n",
+    "import pandas as pd\n",
+    "import numpy as np\n",
+    "import xgboost as xgb\n",
+    "from xgboost import XGBClassifier\n",
+    "from secrets import token_bytes\n",
+    "from pyope.ope import OPE\n",
+    "from sklearn.model_selection import train_test_split"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {
+    "pycharm": {
+     "name": "#%% md\n"
+    }
+   },
+   "source": [
+    "# XGBoost for Dataset\n",
+    "\n",
+    "(We use this example to demenstrate how to use ppxgboost for encypting an xgboost model and query it.)\n",
+    "\n",
+    "The sample dataset we are going to use in this lab is a sampled version of the \"Diabetes 130-US hospitals for years 1999-2008 Data Set\"  (Beata Strack, Jonathan P. DeShazo, Chris Gennings, Juan L. Olmo, Sebastian Ventura, Krzysztof J. Cios, and John N. Clore, “Impact of HbA1c Measurement on Hospital Readmission Rates: Analysis of 70,000 Clinical Database Patient Records,” BioMed Research International, vol. 2014, Article ID 781670, 11 pages, 2014. ).\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "pycharm": {
+     "name": "#%% md\n"
+    }
+   },
+   "source": [
+    "### Data Preparation and Train an XGBoost ML model"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "# In the following example, the datasets modified to match the input requirements by SageMaker Data Wrangler.\n",
+    "data = pd.read_csv('../data/readmitted.csv')\n",
+    "\n",
+    "train, test = train_test_split(data, train_size = 0.002, test_size = 0.002)\n",
+    "\n",
+    "# Training dataset\n",
+    "X_train = train[['race', 'gender', 'age', 'time_in_hospital', 'num_lab_procedures', 'num_procedures', 'num_medications', 'number_outpatient', 'number_emergency', 'number_inpatient', 'number_diagnoses', 'max_glu_serum', 'a1c_result', 'change', 'diabetes_med']]\n",
```

Comment:
Nit: this list is duplicated on line 96. Give the list a name and reuse it rather than rewriting it.